core/camera/src/main/java/com/google/jetpackcamera/core/camera/CameraSession.kt

```diff
@@ -182,7 +244,14 @@ internal suspend fun processTransientSettingEvents(
                 }
             }
         }
+        // todo(kc): Enable torch toggle while recording in progress
+        if (currentCameraState.value.videoRecordingState is VideoRecordingState.Active &&
```

Comment:
nit: The `VideoRecordingState` only switches to `Active` once a `VideoRecordEvent.Start` event is received. This means the torch won't turn on until _after_ the recording is in progress. We may get a few frames of the torch turning on in that case. We could probably turn on the torch sooner than that, but you won't be able to go off the `VideoRecordingState`, unless you do something like add a new "Starting" state.